smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -179,7 +192,11 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         require(sigs.length == claims.length, "INV_CLM_LEN");
         require(sigs.length == signatureData.length, "INV_SIG_LEN");
 
+        uint256 intermediateNonce = lastNonceSubmitted;
+        lastNonceSubmitted += claims.length;
```

Comment:
Let's merge this code in on the other branch so this change doesn't show on this PR.